functions/confs.js

```diff
@@ -11,7 +11,9 @@ exports.init = (client) => {
   // Load default configuration, create if not exist.
   defaultConf = {
     prefix: {type: "String", data: client.config.prefix},
-    disabledCommands: {type: "Array", data: "[]"}
+    disabledCommands: {type: "Array", data: []},
+    mod_role: {type: "String", data: "Mods"},
```

Comment:
Instead of hard coding Mods and Devs to be default

Why not default them as blank values?